ClemBot.Bot/bot/cogs/emote_board_cog.py

```diff
@@ -0,0 +1,430 @@
+                channel_mentions.append(c.mention)
+            return "Channels", "\n".join(channel_mentions)
+
+    async def _get_board(
+        self, eb: EMOTEBOARD_TYPE, ctx: ext.ClemBotCtx, send_error: bool = True
+    ) -> EmoteBoard | None:
+        """
+        Gets the emote board with the following name or emote that belongs to the guild.
+        Checks against both the emote board name and emote for consistency.
+        Returns None if a board with the given name or emote does not exist in the guild.
+        """
+        value = (eb if isinstance(eb, str) else str(eb)).casefold()
+        boards = await self.bot.emote_board_route.get_emote_boards(ctx.guild)
+        for name, emote in boards.items():
+            if name.casefold() == value or emote.casefold() == value:
+                return await self.bot.emote_board_route.get_emote_board(
+                    ctx.guild, name, raise_on_error=True
+                )
+
+        if send_error:
+            await self._error_embed(
+                ctx,
+                f'An emote board with the given {"emote" if self._is_emoji(eb) else "name"} '
+                f'{eb if self._is_emoji(eb) else f"`{eb}`"} could not be found.',
+            )
+
+        return None
+
+    async def _error_embed(self, ctx: ext.ClemBotCtx, description: str) -> None:
+        """Shorthand for sending an error message w/ consistent formatting."""
+        embed = discord.Embed(title="Error", color=Colors.Error)
+        embed.description = description
+        embed.set_footer(text=str(ctx.author), icon_url=ctx.author.display_avatar.url)
+        message = await ctx.send(embed=embed)
+        await self.bot.messenger.publish(
+            Events.on_set_deletable, msg=message, author=ctx.author, timeout=60
+        )
+
+    def _is_emoji(self, s: Union[str, discord.Emoji, discord.PartialEmoji]) -> bool:
+        """
+        Checks if either the given type is `discord.Emoji`,
+        or if the type is `str`, check if it's a valid unicode emoji or starts and ends with `:`.
+        """
+        return (
+            isinstance(s, discord.Emoji | discord.PartialEmoji)
+            or emoji.is_emoji(s)
+            or (s.startswith(":") and s.endswith(":"))
+        )
+
+    def _chunked_boards(
+        self, boards: dict[str, str], n: int, ctx: ext.ClemBotCtx, title: str
+    ) -> list[discord.Embed]:
+        """Chunks the given boards into a markdown-ed list of n-sized items (row * col)"""
+        pages = []
+        boards_list = [
+            (name, emote) for name, emote in boards.items()
+        ]  # flatten the dict into an index-based list
+        for chunk in chunk_sequence(boards_list, n):
+            embed = discord.Embed(title=title, color=Colors.ClemsonOrange)
+            embed.description = f"Here are the emote boards for **{ctx.guild.name}**"
+            for name, emote in chunk:
+                embed.add_field(name=name, value=emote)
```

Comment:
It could be possible to rewrite what the route returns but this route was made specifically to only cache the guild's board's name and emote only (this is also used on the bot-side as a lookup for `emote -> name`). Definitely wouldn't be a bad idea, though... I could possibly also see some kind of total post count working as well.